app/assets/stylesheets/components/common/_news_signup_inpage.scss

```diff
@@ -0,0 +1,531 @@
+$mq-news-signup-inpage-article: px(550);
+$component-border-radius: 5px;
+
+.news-signup-inpage {
+  @extend %stripe-bg;
+  @include clearfix;
+  position: relative;
+  border-radius: $component-border-radius;
+
+  form.is-errored {
+    border-left: 0;
+    padding-left: 0;
+  }
+
+  .news-signup-inpage__header {
+    @include clearfix;
+    display: block;
+    position: relative;
+
+    .news-signup-inpage__text-container {
+      position: relative;
+      margin-left: 0;
+
+      @include respond-to($mq-m) {
+        margin-left: 140px;
+      }
+    }
+
+    .news-signup-inpage__title {
+      @include body(16, 22);
+      color: $color-heading-extra-small;
+      text-align: center;
+      margin: 10px 170px 25px 20px;
+      font-weight: 400;
+      min-height: 130px;
+      position: relative;
+
+      @include respond-to($mq-s) {
+        @include body(18, 24);
+        margin: 0 170px 10px 20px;
+        min-height: 110px;
+      }
+
+      @include respond-to($mq-m) {
+        @include body(16, 23);
+        margin: 20px 10px 10px 10px;
+        min-height: 75px;
+      }
+
+      @include respond-to($mq-3col-layout) {
+        @include body(16, 21);
+        margin: 20px 20px 10px 20px;
+      }
+
+      @include respond-to($mq-l) {
+        @include body(18, 24);
+        margin: 20px 20px;
+      }
+
+      @include respond-to($mq-xl) {
+        @include body(20, 24);
+        margin: 20px 40px 15px 40px;
+      }
+
+      span {
```

Comment:
Would be good to try an use a classname for this rather than the element directly